Tests/QueuesFluentDriverTests/QueuesFluentDriverTests.swift

```diff
@@ -237,6 +245,105 @@ final class QueuesFluentDriverTests: XCTestCase {
         )
     } }
 
+    func testOldFormatMigration() async throws { try await self.withEachDatabase(tableName: nil) {
+        // N.B.: One probably notices that MySQL gets a lot of special-casing in this test. This is another instance
+        // of it; we only need this so the `SET time_zone` thing will work.
+        try await self.app.db(self.dbid).withConnection { db in
```

Comment:
I think I know what you're going for here but indentation still seems weird. Maybe we should standardise the repo  with `swift-format` too

Reply:
The indentation is weird, because otherwise every single test would be indented an extra level.